### DISCUSSION TITLE
Grid showing fault, Search node process status unknown

### DISCUSSION THREAD

**Discussion by ArchiB1** [Security-Onion-Solutions/securityonion#7250] [Unsupported Versions]
After installing the searchnode an alert popped up in Grid. The Search node has a status of "fault" and the process status is unknown. The sensors installed are all showing OK for the process status. The manager is also able to communicate with the Searchnode and run salt commands. So-status returns with no errors. The fault error has no other indications for what could be wrong. Grafana is also unable to populate any information about the Searchnode. We are new to Security Onion and wondering what is causing the fault and what processes are not being sent to allow the manager to see the status.

**Comment by dougburks** [comment]
Are there any network firewalls that might be preventing the search node from fully connecting to the manager?
https://docs.securityonion.net/en/2.3/firewall.html

**Reply by ArchiB1** [reply]
No, and all the other sensors are connected with no issues. After checking the logs, the error message from /opt/so/log/soc on the manager is reading "Host not found in process status metrics".  I am trying to locate if there is possibly a config file with a value set to false for sending this information.

**Reply by dougburks** [reply]
Just to make sure that this not a network connection issue, please try connecting from the search node to the manager on port 443 (and the other ports listed at https://docs.securityonion.net/en/2.3/firewall.html#node-communication) using `nc` or similar tool.
